pkg/controller/template/render.go

```diff
@@ -124,43 +122,12 @@ func GenerateMachineConfigsForRole(config *RenderConfig, role, templateDir strin
 		if err != nil {
 			return nil, err
 		}
-		// defaulting the CGroups version to "v1"
```

Comment:
I still feel like we should update this to be hardcoded to v2 so rhel 8 workers have the default version of v2...

Reply:
Sure. Noted

Reply:
Addressed. 
Adding the code here would update the cgroupsv2 related kernelArgs to the base templates. If the cluster upgrades from 4.13, the cgroupsv1 related kernelArgs are added to `97-{master/worker}-generated-kubelet` machineconfigs. When the rendering happens where both the cgroup related kernelArgs are appended, the cluster might go into unstable state.
Hence, I added the code in the `kubelet_config_nodes.go`